src/recval/evaluator.py

```diff
@@ -3,28 +3,30 @@
 import logging
 from dataclasses import dataclass, field
 
-import numpy as np
+import numpy
```

Comment:
Why the alias for numpy and pandas libraries have been removed?

Reply:
Perche' sono da analisti che lavorano sui notebook, se scrivi una libreria che qualcuno deve usare e' importante esplicitare le dipendenze